collector/systemd_linux.go

```diff
@@ -112,6 +113,11 @@ func NewSystemdCollector(logger log.Logger) (Collector, error) {
 		"Whether the system is operational (see 'systemctl is-system-running')",
 		nil, nil,
 	)
+	systemShutdownDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "system_shutdown_timestamp"),
+		"Time for a scheduled shutdown (see 'systemctl status systemd-shutdownd.service')",
```

Comment:
that command outputs:

```
Unit systemd-shutdownd.service could not be found.
```

here.

I found a `systemd-shutdown(8)` manual page, that doesn't much in understanding that component. In fact, I had a frustrating time trying to find *any* meaningful documentation on how that damn thing works... The [org.freedesktop.login1(5)](https://manpages.debian.org/org.freedesktop.login1) manual page does mention it though:

```
       ScheduledShutdown shows the value pair set with the
       ScheduleShutdown() method described above.
```

That's for the property we're (trying to) fetch(ing) here... That method referenced there is:

```
       ScheduleShutdown() schedules a shutdown operation type at time
       usec in microseconds since the UNIX epoch.  type can be one of
       "poweroff", "dry-poweroff", "reboot", "dry-reboot", "halt", and
       "dry-halt". (The "dry-" variants do not actually execute the
       shutdown action.)  CancelScheduledShutdown() cancels a scheduled
       shutdown. The output parameter cancelled is true if a shutdown
       operation was scheduled.
```

... which is, frankly, not that much helpful.